mass_mailing_event_registration_exclude/data/event_state_data.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
```

Comment:
```suggestion
<odoo noupdate="1">
```

Reply:
@victoralmau done. Good catch!

Reply:
Sorry, why `noupdate`? The attendee states won't change.